.github/workflows/test.yml

```diff
@@ -83,3 +88,17 @@ jobs:
         run: cover -report codecov
         env:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+      - name: Install precious
```

Comment:
Don't we want this to be in a separate job, so it's shown as a separate status check?